front-end/src/app/cruds/pacientes/create-paciente/create-paciente.component.html

```diff
@@ -64,9 +65,13 @@
             <label for="email">E-mail</label>
             <input id="email" [(ngModel)]="email" type="text" pInputText>
         </div>
- <div class="p-formgrid p-grid">
-            <button pButton type="button" class="p-button-raised p-button-success" label="Salvar" icon="pi pi-check" (click)="salvar()"></button>
+        <div class="p-fluid p-formgrid p-grid p-field p-col-1 p-md-1 p-lg-1">
+            <button pButton type="button" class="p-button-raised p-button-success" label="Salvar" icon="pi pi-check" (click)="salvar()" style="top: 10px;"></button>
+        </div>
+         <div class="p-fluid  p-grid p-field p-col-2 p-md-2 p-lg-2">
+          <a href="http://localhost:4200/">  <button pButton type="button" class="p-button-danger" label="Cancelar" icon="pi pi-times" style="left:50px; top: 10px;"></button></a>
```

Comment:
```suggestion
          <button pButton type="button" class="p-button-danger" [routerLink]="['/home']" label="Cancelar" icon="pi pi-times" style="left:50px; top: 10px;"></button></a>
```

Usa routerLink em vez de href, se mudar o endereço do site vc teria q mudar aqui tambem